src/device.cpp

```diff
@@ -645,7 +645,8 @@ void cvk_device::build_extension_ils_list() {
         m_extensions.push_back(
             MAKE_NAME_VERSION(1, 0, 0, "cl_khr_device_uuid"));
         VkSubgroupFeatureFlags required_subgroup_ops =
-            VK_SUBGROUP_FEATURE_BASIC_BIT | VK_SUBGROUP_FEATURE_ARITHMETIC_BIT;
+            VK_SUBGROUP_FEATURE_BASIC_BIT | VK_SUBGROUP_FEATURE_ARITHMETIC_BIT |
+            VK_SUBGROUP_FEATURE_VOTE_BIT | VK_SUBGROUP_FEATURE_BALLOT_BIT;
```

Comment:
`VK_SUBGROUP_FEATURE_VOTE_BIT` and `VK_SUBGROUP_FEATURE_BALLOT_BIT` are not required for basic OpenCL subgroup support. Let's remove them from there so we enable basic subgroup support on more Vulkan implementations.

Reply:
Done.